### ISSUE TITLE
Package regressions for 8.2 release

### ISSUE THREAD

**Issue by samth** [racket/racket#3896]
tl;dr: Currently in good shape.

### Build Failures
- [x] cs-bootstrap https://plt.eecs.northwestern.edu/pkg-build/server/built/fail/cs-bootstrap.txt (repaired already in cs-bootstrap)
- [x] satore https://plt.eecs.northwestern.edu/pkg-build/server/built/fail/satore.txt (same case-sensitivity problem as last time)

### Test Failures
- [x] pie https://plt.eecs.northwestern.edu/pkg-build/server/built/test-fail/pie.txt (appears to be an error message source location printing change from NN.NN to NN:NN). Has an open PR to fix.

### Conflicts
- [ ] video-v0-0 (I believe @LeifAndersen makes these packages conflict intentionally)
- [ ] syndicate-classic (conflicts with `syndicate`, this is not a regression in reality, `syndicate` has a transient build failure on pkg-build.racket-lang.org)

**Comment by rfindler** [comment]
The first pkg build with the release candidate has completed: https://plt.cs.northwestern.edu/release-pkg-build/

**Comment by samth** [comment]
I've updated with the release pkg-build results. We're still in good shape.
